randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2052,15 +2056,20 @@ Extra - asset_id: collision_camera_026
   * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
   * Extra - actor_layer: breakables
   * Extra - tile_types: ('WEIGHT',)
-  > Door to Cross Bomb Tutorial
-      Before Elun - Release X Parasites
   > Before Golzuna
       After Elun - Release X Parasites
 
 > Start Point; Heals? False; Spawn Point
   > Door to Cross Bomb Tutorial
       Trivial
 
+> Dock from Above Golzuna; Heals? False
+  * Blocked Passage to Above Golzuna/Dock to Golzuna Arena
```

Comment:
Can you add a description to this node and the other dock, so it's clear it's for an SSC?

Reply:
Added some descriptions